.travis.yml

```diff
@@ -1,8 +1,5 @@
 language: php
 php:
-  - '5.5'
```

Comment:
Support for these versions should not be dropped.

Reply:
Its right to drop support for php versions <= 7.0 because they aren't in active support anymore.  Even if 5.6 and 7.0 get security fixes, reach their end of life on Dec 2018.
See [php.net/supported-versions](http://php.net/supported-versions.php) for more.

Reply:
I think that since this is Symfony-related, version constraints should be aligned with constraints Symfony enforces. If this version supports Symfony 3.4, it should support same php versions as Symfony 3.4 (5.5+)

Reply:
>Its right to drop support for php versions <= 7.0

IMO, it dependens which version this PR will affect. If `nostromo` is meant for `3.x` branch and `master` will alias `4.x`, I agree that we can accept non BC changes like this. If that is the case, I'll be happy to submit a PR for the current latest stable version (`nostromo`) allowing symfony 4 support.